src/css/app.scss

```diff
@@ -1,7 +1,11 @@
 @import './colors.variables.scss';
+@import './fontFamilies.scss';
 
-// app global css in SCSS form
+* {
+  font-family: 'Visby';
+}
 
+// app global css in SCSS form
```

Comment:
I think this comment is not needed as is not too much informative

Reply:
Thanks! I don't remember why I put it there

Reply:
Thanks! I forgot why I posted that comment.